.github/workflows/build-and-validate.yaml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build-validate:
     name: Build and validate
-    runs-on: ubuntu-18.04
```

Comment:
I see `latest` and `20.04` - which one should we use?

Reply:
some actions will run fine on ubuntu-latest, others such as `setup-ruby` require ubunutu-20.04, hence the mix.